README.md

```diff
@@ -38,109 +56,125 @@ No faster way to get started than by diving in and playing around with a demo.
 
 Need quickstarts to begin your Redis AI journey? **Start here.**
```

Comment:
I think the start here part is a little odd when reading I was trying to click it like it was a link.